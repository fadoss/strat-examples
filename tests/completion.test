deduction/completion/examples/example-6.2.7

srew < eqs, mtRlS > using compl .
srew < eqs, mtRlS > using N-COMP .
srew < eqs, mtRlS > using S-COMP .
srew < eqs, mtRlS > using ANS-COMP .

load example-7.1.1

dsrew [1] < eqs, mtRlS > using S-COMP .

load example-groups

dsrew [1] < eqs, mtRlS > using ANS-COMP .
