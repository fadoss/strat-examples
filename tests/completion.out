==========================================
srewrite in EX627-STRAT : < eqs,mtRlS > using compl .

Solution 1
rewrites: 6750
result System: < mtEqS,'f['f['x:S]] -> 'g['x:S] 'f['g['x:S]] -> 'g['f['x:S]] >

No more solutions.
rewrites: 6750
==========================================
srewrite in EX627-STRAT : < eqs,mtRlS > using N-COMP .

Solution 1
rewrites: 2271
result System: < mtEqS,'f['f['x:S]] -> 'g['x:S] 'f['g['x:S]] -> 'g['f['x:S]] >

No more solutions.
rewrites: 2271
==========================================
srewrite in EX627-STRAT : < eqs,mtRlS > using S-COMP .

Solution 1
rewrites: 2244
result System: < mtEqS,'f['f['x:S]] -> 'g['x:S] 'f['g['x:S]] -> 'g['f['x:S]] >

No more solutions.
rewrites: 2244
==========================================
srewrite in EX627-STRAT : < eqs,mtRlS > using ANS-COMP .

Solution 1
rewrites: 2253
result System: < mtEqS,'f['f['x:S]] -> 'g['x:S] 'f['g['x:S]] -> 'g['f['x:S]] >

No more solutions.
rewrites: 2253
==========================================
dsrewrite [1] in EX711-STRAT : < eqs,mtRlS > using S-COMP .

Solution 1
rewrites: 18613
result System: < mtEqS,'*['y:S,'*['*['y:S,'z:S],'z1:S]] -> '*['y:S,'z:S] '*['*[
    'x1:S,'*['x:S,'y:S]],'y:S] -> '*['x:S,'y:S] '*['*['x:S,'y:S],'*['y:S,'z:S]]
    -> 'y:S >
==========================================
dsrewrite [1] in EX74-STRAT : < eqs,mtRlS > using ANS-COMP .

Solution 1
rewrites: 94965
result System: < mtEqS,'*['e.S,'x:S] -> 'x:S '*['y1:S,'*['I['y1:S],'z3:S]] ->
    'z3:S '*['y1:S,'I['y1:S]] -> 'e.S '*['z2:S,'e.S] -> 'z2:S '*['*['x:S,'y:S],
    'z:S] -> '*['x:S,'*['y:S,'z:S]] '*['I['x:S],'x:S] -> 'e.S '*['I['y1:S],'*[
    'y1:S,'z1:S]] -> 'z1:S 'I['e.S] -> 'e.S 'I['*['y4:S,'y6:S]] -> '*['I[
    'y6:S],'I['y4:S]] 'I['I['y1:S]] -> 'y1:S >
Bye.
